scene/debugger/scene_debugger.h

```diff
@@ -295,6 +295,9 @@ class RuntimeNodeSelect : public Object {
 	void _clear_selection();
 	void _set_selection_visible(bool p_visible);
 
+	void _open_selection_list(Vector<SelectResult> items, Point2 pos);
```

Comment:
We use `const T &` for these types where possible.

```suggestion
	void _open_selection_list(Vector<SelectResult> items, const Point2 &pos);
```

Reply:
Also for the `Vector`, and prefix arguments with `p_`
```suggestion
	void _open_selection_list(const Vector<SelectResult> &p_items, const Point2 &p_pos);
```